packages/legend-query-builder/style/_query-builder.scss

```diff
@@ -527,9 +527,13 @@
 
   &__item__action {
     margin-left: 0.5rem;
+    display: flex;
```

Comment:
Can you make sure this doesn't break the styling for the "Show in tree" button on the fetch structure column info tooltip?

Reply:
<img width="162" alt="Screenshot 2024-08-02 at 4 22 34 PM" src="https://github.com/user-attachments/assets/90f6c8fc-adf4-48fd-bdd0-eddfedf42db7">

It will help align this icon too